Networking/Networking/Model/WCPayCardPresentReceiptDetails.swift

```diff
@@ -14,14 +14,14 @@ public struct WCPayCardPresentReceiptDetails: Codable, GeneratedCopiable, Genera
     public let accountType: WCPayCardFunding
 
     /// The EMV Application Identifier (AID)
-    public let applicationPreferredName: String
+    public let applicationPreferredName: String?
 
     /// The EMV Dedicated File (DF) Name
-    public let dedicatedFileName: String
+    public let dedicatedFileName: String?
 
     public init(accountType: WCPayCardFunding,
-                applicationPreferredName: String,
-                dedicatedFileName: String) {
+                applicationPreferredName: String?,
+                dedicatedFileName: String?) {
```

Comment:
Since this is blocking other work, I'll go ahead and 👍🏽 but I'm a bit uncomfortable with these optionals. Can we add a comment to make sure we get back to this when we know more from Stripe?